src/query/query_hash_scan.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2016 CUBRID Corporation
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 2 of the License, or
+ *   (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ *
+ */
+
+//
+// query_hash_scan - interface for hash list scan during queries
+//
+
```

Comment:
please add include guard in header file.

```suggestion

#ifndef _QUERY_HASH_SCAN_H_
#define _QUERY_HASH_SCAN_H_

```

Reply:
I fixed it.